source/type/tests/comments_test.cpp

```diff
@@ -50,7 +50,7 @@ BOOST_GLOBAL_FIXTURE(logger_initialized);
 
 namespace navitia {
 namespace type {
-bool operator==(const std::vector<Comment>& lhs, const std::vector<std::string>& rhs) {
+static bool operator==(const std::vector<Comment>& lhs, const std::vector<std::string>& rhs) {
```

Comment:
Why ? No need `static`, because we are on a **namespace**
In addition to that, i don't know where it is used

Reply:
Just remove a clang warning, as it's missing prototype.